src/bundled_files/wpilib_preferences.json.ts

```diff
@@ -0,0 +1,10 @@
+import { unindent } from "../codegen/java/util"
+
+export const BundledPreferences = unindent(`
+  {
+    "enableCppIntellisense": false,
+    "currentLanguage": "java",
+    "projectYear": "2025",
```

Comment:
In the future, this will want to be a global variable somewhere shared, instead of being hardcoded